packages/editor/src/client/panel/memberPlanEditPanel.tsx

```diff
@@ -106,6 +106,7 @@ export function MemberPlanEditPanel({id, onClose, onSave}: MemberPlanEditPanelPr
 
   useEffect(() => {
     if (data?.memberPlan) {
+      if (!data.memberPlan.tags) data.memberPlan.tags = ['']
```

Comment:
Why did you add this?

Reply:
My IDE does weird stuff... it changed something else which made this change required have fixed this now